eclass/python-utils-r1.eclass

```diff
@@ -1492,11 +1492,7 @@ eunittest() {
 	_python_check_occluded_packages
 
 	# unittest fails with "no tests" correctly since Python 3.12
-	local runner=unittest
-	if _python_impl_matches "${EPYTHON}" 3.{9..11}; then
-		runner=unittest_or_fail
-	fi
-	set -- "${EPYTHON}" -m "${runner}" discover -v "${@}"
+	set -- "${EPYTHON}" -m unittest discover -v "${@}"
```

Comment:
In such case, let's also drop >=3.12 COMPAT from unittest-or-fail package.

Reply:
We will be last riting it instead.